packages/core/drizzle/0164_purple_warbound.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "latitude"."workspaces" ADD COLUMN "uuid" uuid DEFAULT gen_random_uuid() NOT NULL;
```

Comment:
this is backfilling existing workspaces?

Reply:
It did when I ran in my dev env